doc/filters/format_currency.rst

```diff
@@ -20,25 +20,75 @@ You can pass attributes to tweak the output:
 
 The list of supported options:
 
-* ``grouping_used``;
-* ``decimal_always_shown``;
-* ``max_integer_digit``;
-* ``min_integer_digit``;
-* ``integer_digit``;
-* ``max_fraction_digit``;
-* ``min_fraction_digit``;
-* ``fraction_digit``;
-* ``multiplier``;
-* ``grouping_size``;
-* ``rounding_mode``;
-* ``rounding_increment``;
-* ``format_width``;
-* ``padding_position``;
-* ``secondary_grouping_size``;
-* ``significant_digits_used``;
-* ``min_significant_digits_used``;
-* ``max_significant_digits_used``;
-* ``lenient_parse``.
+* ``grouping_used``: Specifies whether to use grouping separator for thousands.
+    .. code-block:: twig
+
+        {# €1,234,567.89 #}
+        {{ 1234567.89 | format_currency('EUR', {grouping_used:true}, 'en') }}
+
+* ``decimal_always_shown``: Specifies whether to always show the decimal part, even if it's zero.
+    .. code-block:: twig
+
+        {# €123.00 #}
+        {{ 123 | format_currency('EUR', {decimal_always_shown:true}, 'en') }}
+
+* ``max_integer_digit``:
+* ``min_integer_digit``:
+* ``integer_digit``: Define constraints on the integer part.
+    .. code-block:: twig
+
+        {# €345.68 #}
+        {{ 12345.6789 | format_currency('EUR', {max_integer_digit:3, min_integer_digit:2}, 'en') }}
+
+* ``max_fraction_digit``:
+* ``min_fraction_digit``:
+* ``fraction_digit``: Define constraints on the fraction part.
+    .. code-block:: twig
+
+        {# €123.46 #}
+        {{ 123.456789 | format_currency('EUR', {max_fraction_digit:2, min_fraction_digit:1}, 'en') }}
+
+* ``multiplier``: Multiplies the value before formatting.
+    .. code-block:: twig
+
+        {# €123,000.00 #}
+        {{ 123 | format_currency('EUR', {multiplier:1000}, 'en') }}
+
+* ``grouping_size``:
+* ``secondary_grouping_size``: Set the size of the primary and secondary grouping separators.
+    .. code-block:: twig
+
+        {# €1,23,45,678.00 #}
+        {{ 12345678 | format_currency('EUR', {grouping_size:3, secondary_grouping_size:2}, 'en') }}
+
+* ``rounding_mode``:
+* ``rounding_increment``: Control rounding behavior.
+    .. code-block:: twig
+
+        {# €123.50 #}
+        {{ 123.456 | format_currency('EUR', {rounding_mode:'ceiling', rounding_increment:0.05}, 'en') }}
+
+* ``format_width``:
+* ``padding_position``: Set width and padding for the formatted number.
+    .. code-block:: twig
+
+        {# €123.00 #}
+        {{ 123 | format_currency('EUR', {format_width:10, padding_position:'before_suffix'}, 'en') }}
```

Comment:
Maybe list the different `padding_position` available ?

Reply:
I added it too